Source/Core/DolphinQt/SkylanderPortal/SkylanderPortalWindow.cpp

```diff
@@ -770,30 +776,13 @@ bool SkylanderPortalWindow::PassesFilter(QString name, u16 id, u16 var)
   bool pass = false;
 
   // Check against active game filters
-  if (m_game_filters[GetGameID(IOS::HLE::USB::Game::SpyrosAdv)]->isChecked())
-  {
-    if (character.game == IOS::HLE::USB::Game::SpyrosAdv)
-      pass = true;
-  }
-  if (m_game_filters[GetGameID(IOS::HLE::USB::Game::Giants)]->isChecked())
-  {
-    if (character.game == IOS::HLE::USB::Game::Giants)
-      pass = true;
-  }
-  if (m_game_filters[GetGameID(IOS::HLE::USB::Game::SwapForce)]->isChecked())
-  {
-    if (character.game == IOS::HLE::USB::Game::SwapForce)
-      pass = true;
-  }
-  if (m_game_filters[GetGameID(IOS::HLE::USB::Game::TrapTeam)]->isChecked())
-  {
-    if (character.game == IOS::HLE::USB::Game::TrapTeam)
-      pass = true;
-  }
-  if (m_game_filters[GetGameID(IOS::HLE::USB::Game::Superchargers)]->isChecked())
+  for (size_t i = 0; i < 5; i++)
   {
-    if (character.game == IOS::HLE::USB::Game::Superchargers)
-      pass = true;
+    if (m_game_filters[i]->isChecked())
```

Comment:
Maybe combine these in to one single if statement

`if (m_game_filters[i]->isChecked() && character.game == (IOS::HLE::USB::Game)i)` or whatever looks best to you